src/resources/js/dependency.js

```diff
@@ -108,11 +108,10 @@
 		}
 
 		$dependents.each( function( k, dependent ) {
-			var $dependent         = $( dependent );
-			var hasDependentParent = $dependent.is( '[data-dependent-parent]' );
+			var $dependent = $( dependent );
 
-			if ( hasDependentParent ) {
-				var dependentParent  = $dependent.data( 'dependentParent' );
+			if ( $dependent.is( '[data-dependent-parent]' ) ) {
+				var dependentParent  = $dependent.data( 'dependent-parent' );
```

Comment:
Do we have any code that depends on this (that uses `data-dependentParent`)? This will break any implementation of that if so.

Reply:
We do not that I could find. Although @bordoni might know of an instance?

Reply:
http://p.tri.be/MLPrhj no `data-dependent-parent (except here and in the js)
http://p.tri.be/WV1eor no `dependentParent` (except in the js)

Reply:
OK. My other worry is if we have cases in the code where we construct these data attributes using variables, like `"data-$key"` or, less likely, `"data-dependent$key"`. But if we're fairly certain this was used for widgets, I'd say let's be cautious and keep an eye out for anything that looks amiss.